libdebug/ptrace/ptrace_interface.py

```diff
@@ -219,6 +219,11 @@ def kill(self: PtraceInterface) -> None:
 
     def cont(self: PtraceInterface) -> None:
         """Continues the execution of the process."""
+
+        # Forward signals to the threads
+        if len(self.context._resume_context.threads_with_signals_to_forward) > 0:
```

Comment:
No need to call len() on the list

Reply:
19b9c1f07508b6fdc0cd6dd4778b4f08696c1109